### ISSUE TITLE
what is the test set you used in your paper

### ISSUE THREAD

**Issue by TysonYu** [tahmedge/QR-BERTSUM-TL-for-QFAS#2]
Hi,
Thank you for providing the code! I have a question about the test data that you use.
Do you also use 10-fold cross-validation in BERTSUM experiments that you reported in the paper? If you use 10-fold cross-validation, you will train 10 models with the different training set and test on 10 test set（with each1,357 samples).
Is my understanding correct？

**Comment by tahmedge** [comment]
Yes. Your understanding is correct. The folds are created based on the official script given here: https://github.com/PrekshaNema25/DiverstiyBasedAttentionMechanism

Read the original paper of this dataset to know more in detail about the dataset: https://arxiv.org/pdf/1704.08300.pdf

**Comment by TysonYu** [comment]
Thank you! @tahmedge
